spec/api/molecule_api_spec.rb

```diff
@@ -117,6 +111,18 @@
           })
         end
       end
+
+      context 'when a molecule with a corrupt molfile is already present', skip: 'present for reproducibility' do
+        it 'cannot creates the correct molecule if one with corrupt molfile is there' do
```

Comment:
Ambiguous Test Description:
Suggestion: it 'does not create a molecule with the same ID as the corrupt one' do

Reply:
thx I will revisit those tests... 
but as discussed... later😅